java-frontend/src/main/java/org/sonar/java/SonarComponents.java

```diff
@@ -86,7 +86,11 @@
   public static final String SONAR_AUTOSCAN_CHECK_FILTERING = "sonar.internal.analysis.autoscan.filtering";
   public static final String SONAR_BATCH_SIZE_KEY = "sonar.java.experimental.batchModeSizeInKB";
   public static final String SONAR_FILE_BY_FILE = "sonar.java.fileByFile";
-  public static final String SONAR_CAN_SKIP_UNCHANGED_FILES_KEY = "sonar.java.internal.skipUnchanged";
+  /**
+   * Enables optimized analysis of unchanged by skipping some checks.
+   * Defaults to true.
+   */
```

Comment:
Technically, this is the override flag, which is `null` when not set. And whether or not we skip unchanged files depends on the context & whether or not SQ/SC supports it. Perhaps the `Defaults to true` could be confusing, as it actually won't always be enabled by default?